cli/src/pcluster/templates/queues_stack.py

```diff
@@ -385,3 +360,51 @@ def _add_compute_resource_launch_template(
         )
 
         return launch_template
+
+
+def add_network_interfaces(
+    queue,
+    compute_resource,
+    queue_lt_security_groups,
+):
+    """Generate launch template network interfaces list"""
+
+    is_gb200 = compute_resource.instance_types[0].split(".")[0] == P6E_GB200
+    interface = "efa" if compute_resource.efa and compute_resource.efa.enabled and not is_gb200 else None
```

Comment:
[minor] Here and below: this is not an interface, but an interface type. What about renaming it to `interface_type`?